python/kwiver/sprokit/util/pystream.cxx

```diff
@@ -6,7 +6,7 @@
 
 #include <pybind11/pybind11.h>
 
-#include <python/kwiver/vital/util/pybind11.h>
+#include <pybind11/pybind11.h>
```

Comment:
Let's collapse this to just one include. I'm not sure yet if there are other artifacts like this yet, but do the same there as well.